gpu/install_gpu_driver.sh

```diff
@@ -37,48 +44,63 @@ function install_gpu_driver() {
     exit 0
   fi
 
-  # Add non-free Debian 9 Stretch packages.
+  local packages=(nvidia-cuda-toolkit)
+  local modules=(nvidia-drm nvidia-uvm drm)
+
+  # Add non-free Debian packages.
   # See https://www.debian.org/distrib/packages#note
-  for type in deb deb-src; do
-    for distro in stretch stretch-backports; do
-      for component in contrib non-free; do
-        echo "${type} http://deb.debian.org/debian/ ${distro} ${component}" \
+  if [[ ${OS_NAME} == debian ]]; then
+    for type in deb deb-src; do
+      for distro in ${OS_DIST} ${OS_DIST}-backports; do
+        echo "${type} http://deb.debian.org/debian ${distro} contrib non-free" \
           >>/etc/apt/sources.list.d/non-free.list
       done
     done
-  done
+    packages+=(nvidia-driver nvidia-kernel-common nvidia-smi)
+    modules+=(nvidia-current)
+    local nvblas_cpu_blas_lib=/usr/lib/libblas.so
+  fi
+
+  if [[ ${OS_NAME} == ubuntu ]]; then
```

Comment:
If the OS is neither Ubuntu nor Debian, nvblas_cpu_blas_lib won't be initialized, it is better to return an error early.

Reply:
Done.